src/components/ArticleEditor/index.js

```diff
@@ -0,0 +1,247 @@
+import React, { Component } from 'react';
+import Dante from 'Dante2';
+import { ImageBlockConfig } from 'Dante2/package/es/components/blocks/image';
+import './ArticleEditor.scss';
+import { VideoBlockConfig } from 'Dante2/package/es/components/blocks/video';
+import { CodeBlockConfig } from 'Dante2/package/es/components/blocks/code';
+import { EmbedBlockConfig } from 'Dante2/package/es/components/blocks/embed';
+import { PlaceholderBlockConfig } from 'Dante2/package/es/components/blocks/placeholder';
+import PropTypes from 'prop-types';
+import M from 'materialize-css';
```

Comment:
Thanks @PaulKariukiRimiru for the review,
however, I feel *M* is more intuitive for someone using the Materialize css documentation, since that is what is used throughout the documentation. What do you think?

![screen shot 2018-11-12 at 04 55 32](https://user-images.githubusercontent.com/25629064/48322161-3addbc00-e637-11e8-9a1d-ede1b2a07fe7.png)

Reply:
It is a bad practice to use single letters as variables, the fact that it is in the documentation does not make it right to use `M` as an import variable, please refactor this to something descriptive of what it actually is, for example, you can use `materialize`

Reply:
Okay. Let me do the changes.